packages/beacon-node/test/unit/api/impl/beacon/blocks/utils.test.ts

```diff
@@ -71,6 +71,13 @@ describe("block api utils", function () {
       expect(dbStub.blockArchive.get).to.be.calledOnceWithExactly(expected);
     });
 
+    it("should resolve justified", async function () {
+      const expected = 0;
+      forkChoiceStub.getJustifiedBlock.returns(expectedSummary);
+      await resolveBlockId(forkChoiceStub, dbStub, "justified").catch(() => {});
```

Comment:
Why the catch all here?

Reply:
After reviewing this with @nazarhussain, we found that without the `catch` all these tests fail here

https://github.com/ChainSafe/lodestar/blob/e1acb85436745ac6535e29ef25b5b9ddd2283ae3/packages/beacon-node/src/api/impl/beacon/blocks/utils.ts#L34